cob_twist_controller/include/cob_twist_controller/limiters/limiter_base.h

```diff
@@ -54,4 +77,27 @@ class LimiterBase
         const LimiterParams& limiter_params_;
 };
 
+/// Cartesian Base class for limiters, defining interface methods.
+class LimiterCartesianBase
+{
+    public:
+        explicit LimiterCartesianBase(const LimiterParams& limiter_params) : limiter_params_(limiter_params)
+        {}
+
+        virtual ~LimiterCartesianBase() {}
+
+        /**
+         * Pure virtual method to mark as interface method which has to be implemented in inherited classes.
+         * The intention is to implement a method which enforces limits to the q_dot_out vector according to
+         * the calculated joint velocities and / or joint positions and or cartesian twists.
+         * @param q_dot_ik The calculated joint velocities vector which has to be checked for limits.
+         * @param q The last known joint positions.
+         * @return Scaled joint velocities vector.
```

Comment:
adjust this text in accordance to the function parameters `v_in`, i.e. remove stuff related to `q_dot_out` and joint velocities/positions...